ds-prep-python-functions/main.py

```diff
@@ -0,0 +1,18 @@
+bool_to_int = lambda value: int(value)
+print(bool_to_int(True))
+get_smaller = lambda a , b: min(a , b)
+print(get_smaller(5 , 10))
+def cube(x): return x**3
+print(cube(5))
+def absolute_difference(a , b): return (a - b)
+print(absolute_difference(5 , 3))
```

Comment:
`absolute_difference` should also return a positive number if `b` is greater than `a`:

---

![image](https://user-images.githubusercontent.com/16143771/148815747-40f6783d-ba81-45c6-847d-3f3313d0b561.png)

---

In general, I'd recommend testing your functions with at least two different sets of inputs to make sure it's adapting well to different inputs.